### PR TITLE
Extract features list from README

### PR DESCRIPTION
**Pull request by mkondratek**
- [ ] publish gifs to marketplace

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,35 +147,9 @@ When file editing is done, you can click the button in the top right corner of t
...
-
-### Multi-repository support
-
-Git Machete supports many git repositories in one project, including both regular repositories and submodules.
-If more than one repository is detected, the selection list will appear at the top of the main plugin window.
-In that case, each repository will have its own `machete` file.
-
-![](docs/multi_repo.gif)
-
+Please see the [features list](FEATURES-LIST.md) for more specific features description.
```
Comment (PawelLipski) [inline README.md:152]: I'd say `feature list`... generally I've observed that plural doesn't sound well in English in the "implicit genitive" (Polish: _lista ficzerów_ where _ficzerów_ is in genitive)

Comment (PawelLipski) [reply README.md:152]: Even Saxon genitive (`...'s` or `...'`) usually looks awkward when appended to a plural noun

File: FEATURE-LIST.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+![](docs/open_git_machete.gif)
+
+
+## Branch graph
+
+For each branch, Git Machete indicates the relation to each of its child branches.
+If the edge between them is **green** that means the child branch is in sync with its parent branch &mdash; in other words, there are no commits in the parent branch that don't belong to the child.
+But if there are some commits in the parent branch that are **not** reachable from the child, then the edge is **red** &mdash; you need to [rebase](#rebase) child branch onto the parent.
+Edge can also be **gray** or **yellow** &mdash; that means respectively the branch was merged to the parent, or a fork point can't be determined automatically
```
Comment (PawelLipski) [inline FEATURE-LIST.md:41]: Pls keep up to date with the updated wording in the base PR (separate sentences for `gray` and `yellow`)

File: scripts/setup-readme-scenarios [hunk 3]
```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -e
+
+sandboxDir=machete-sandbox
+
+create_branch() {
```
Comment (PawelLipski) [inline scripts/setup-readme-scenarios:7]: Whoops... could you somehow reuse the existing helpers?...

**Comment by mkondratek** [comment]
@PawelLipski, @micpiotrowski feel free to take a look onto the new gifs 

Leaving this PR as a draft as descriptions are still missing

**Comment by micpiotrowski** [comment]
About GIFs:
I'm not sure if this circular cursor movement are good (my first impression was bad). I'd leave it only when graph edges are indicated and definitely not when you pointing clicked button or just empty space (on first GIF).
Also in 3rd GIF, Im not sure if it would be better if show also a commit creation but maybe README description will be enough.
Speed is good.

File: scripts/setup-readme-scenarios [hunk 4]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
```
Comment (micpiotrowski) [inline scripts/setup-readme-scenarios:1]: :heart: :ok_hand: :heart:

**Comment by PawelLipski** [comment]
Unrel from the actual PR content: the [build](https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/3505/workflows/abf168ff-eb34-44e6-8e90-5014d6682374/jobs/3676/) failed on socket response timeout when calling JS `ide.getProgressIndicators()`... no idea why there specifically, there's no blocking operation happening there :frowning_face:

**Comment by mkondratek** [comment]
> About GIFs:
> I'm not sure if this circular cursor movement are good (my first impression was bad). I'd leave it only when graph edges are indicated and definitely not when you pointing clicked button or just empty space (on first GIF).
> Also in 3rd GIF, Im not sure if it would be better if show also a commit creation but maybe README description will be enough.
> Speed is good.

These concerns have been taken under consideration while we were discussing the gifs f2f with @PawelLipski.
We have decided to leave the gifs as they are for now. 

I agree that mouse movements may look a bit strange. 
We must remember that the descriptions will make them more reasonable and clarify the things. 
Additionally the "mouse rounds" slow down the shown action (which is smth we want).

**Review by PawelLipski — COMMENTED** [review]
Some fixes pushed, but I'll need another pass through the text...

**Comment by PawelLipski** [comment]
Another pass done... I think there's still one left to go

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**